platform/mellanox/issu-version.mk

```diff
@@ -5,5 +5,6 @@ $(ISSU_VERSION_FILE)_SRC_PATH = $(PLATFORM_PATH)/issu-version
 $(ISSU_VERSION_FILE)_DEPENDS += $(APPLIBS)
 SONIC_MAKE_FILES += $(ISSU_VERSION_FILE)
 
-export ISSU_VERSION_FILE
+MLNX_FILES += $(ISSU_VERSION_FILE)
 
+export ISSU_VERSION_FILE
```

Comment:
Will you also remove this line?

Reply:
@qiluo-msft please have a look at: https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/sonic_debian_extension.j2#L465